kuwala/pipelines/population-density/src/Processor.py

```diff
@@ -46,7 +47,23 @@ def start(files: [dict], output_dir: str):
 
         df = reduce((lambda d1, d2: d1.join(d2, ['h3Index'], 'full').repartition(number_of_partitions, 'h3Index')), dfs)
 
-        df.write.mode('overwrite').parquet(output_dir + 'result.parquet')
+        today=str(dte.today())
+
+        if (os.path.isdir(output_dir + 'result.parquet'+'_'+today)):
+            print("parquet files already exist")
+            deci=''
```

Comment:
Would be better if you can use a clear variable name
```suggestion
            decision = ''
```

Reply:
Nice catch @Arun-kc  ! Thank you ^^